spec/integration/probes/elasticsearch_spec.rb

```diff
@@ -24,17 +24,21 @@
     end
 
     it "handles uninitialized probe dependencies" do
+      # Temporarily uninstall NetHTTP and HTTPClient probes
       TempNetHTTPProbe    = Skylight::Probes::NetHTTP::Probe
       TempHTTPClientProbe = Skylight::Probes::HTTPClient::Probe
       Skylight::Probes::NetHTTP.send(:remove_const, :Probe)
       Skylight::Probes::HTTPClient.send(:remove_const, :Probe)
+      allow_any_instance_of(::Net::HTTP).to receive(:request){|obj, *args| obj.request_without_sk(*args)}
+      allow_any_instance_of(::HTTPClient).to receive(:do_request){|obj, *args| obj.do_request_without_sk(*args)}
```

Comment:
If this isn't clear, I suggest you check out my branch and try removing the lines where you commented, you'll see the tests fail and it will make sense what I'm doing.